FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c

```diff
@@ -144,7 +139,7 @@
  * absence of sending any other control packets, the client MUST send a
  * PINGREQ Packet.
  */
-#define mqttexampleKEEP_ALIVE_TIMEOUT_SECONDS        ( 60U )
+#define mqttexampleKEEP_ALIVE_TIMEOUT_SECONDS        ( 15U )
```

Comment:
So before we reported a keep alive time of 60 seconds and ran the timer every 15 seconds, and now we report a time of 15 seconds and run the timer every 3 seconds? Is this to speed up the demo?

Reply:
Right, just to speed-up the demo as everything is a function of this macro. The timer ran for 1/4th of this timeout though.